app/models/manageiq/providers/kubernetes/container_manager/metrics_capture.rb

```diff
@@ -1,10 +1,60 @@
+require 'hawkularclient'
```

Comment:
Actually, this should not be required here...it should be where it's used in the CaptureContext class.